flipperdevices/flipperzero-firmware
# NFC magic cards support

# What's new

- Add initial support for Magic cards
- Add detect Magic card
- Add Write Gen 1A Magic card option. Works with MFC 1k with 4 bytes UIDs
- Add Wipe Magic card command

# Verification 

- Verify Magic cards detection works correctly
- Verify writing suitable MFC cards works correctly

# Checklist (For Reviewer)
